frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -497,6 +497,11 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
                         searchPlaceholder: 'Replay',
                         type: TaxonomicFilterGroupType.Replay,
                         render: ReplayTaxonomicFilters,
+                        valuesEndpoint: (key) => {
+                            if (key === 'visited_page') {
+                                return 'api/event/values/?key=' + encodeURIComponent('$current_url')
```

Comment:
(are we able to|should we) restrict this further to just $pageview events?

Reply:
Yep, good call. I think we can add `&event_names=$pageview`